assets/js/screens/Work/Work.jsx

```diff
@@ -158,6 +160,16 @@ const ScreensWork = () => {
                       </Link>
                     </dd>
                   </dl>
+
+                  {!data.work.collection && (
+                    <p className="notification has-text-centered">
+                      <FontAwesomeIcon icon="exclamation-triangle" /> A work
+                      must belong to a Collection in order to be published.
+                      Click on the{" "}
+                      <a href="#tab-administrative">Administrative tab</a>{" "}
+                      below, to add this work to a Collection.
```

Comment:
`#tab-administrative` navigates to About tab and does not open Administrative Tab.

Reply:
Yea, it's not supposed to open the Administrative tab, but rather just scroll down to that area of the page.  Maybe it's confusing to add the link here?  I was trying to get fancy:)

Reply:
It's only confusing as on my browser, it is scrolling up to `Core Metadata` box and not scrolling to `Tabs`

Reply:
Ah, ok I think I know a fix to make it a little cleaner....

Reply:
The navbar header is getting in the way, and there's nothing else close above in the UI to attach an `id` to which we can use as an anchor target, so just removing for now.  Less confusion is probably better.